Source/WebCore/dom/Node.cpp

```diff
@@ -1993,14 +1996,14 @@ static void traverseTreeAndMark(const String& baseIndent, const Node* rootNode,
 
 void Node::showTreeAndMark(const Node* markedNode1, const char* markedLabel1, const Node* markedNode2, const char* markedLabel2) const
 {
-    const Node* rootNode;
-    const Node* node = this;
+    RefPtr<const Node> rootNode;
+    RefPtr<const Node> node = this;
     while (node->parentOrShadowHostNode() && !node->hasTagName(bodyTag))
         node = node->parentOrShadowHostNode();
-    rootNode = node;
+    rootNode = WTFMove(node);
```

Comment:
Would be nice to use a lambda so we don't have a hanging moved-out-of `node` variable.

Reply:
Will do.